pkg/getter/getter.go

```diff
@@ -183,16 +185,26 @@ const (
 	// The cost timeout references curl's default connection timeout.
 	// https://github.com/curl/curl/blob/master/lib/connect.h#L40C21-L40C21
 	// The helm commands are usually executed manually. Considering the acceptable waiting time, we reduced the entire request time to 120s.
-	DefaultHTTPTimeout = 120
+	defaultHTTPTimeout = 120
 )
 
-var defaultOptions = []Option{WithTimeout(time.Second * DefaultHTTPTimeout)}
```

Comment:
I'm not sure you want to make`GetDfaultHTTPTimeout` and `DefaultOptions` public unless there is a need.

Reply:
My previous idea was that if a user's project depends on helm and wants to get the default http timeout, helm should provide a method to get it, so I set the `GetDefaultHTTPTimeout` method to public.
And there is no need to set the `DefaultOptions` method to public, I'll Update it.